packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -200,6 +201,11 @@ class CupertinoDialogAction extends StatelessWidget {
   /// If this is set to null, the button will be disabled.
   final VoidCallback onPressed;
 
+  /// Set to true if button is the default choice in the dialog.
+  ///
+  /// Default buttons are bolded.
```

Comment:
This is an odd semantic. Does it not do anything else?

Maybe we should call it "recommended"? Does iOS call it "default"?